nautobot/extras/tests/test_api.py

```diff
@@ -2758,7 +2732,7 @@ class RelationshipAssociationTest(APIViewTestCases.APIViewTestCase):
     def setUpTestData(cls):
         cls.location_type = ContentType.objects.get_for_model(Location)
         cls.device_type = ContentType.objects.get_for_model(Device)
-        cls.status_active = Status.objects.get(slug="active")
+        cls.status = Status.objects.first()
```

Comment:
Probably need to distinguish between `Status.objects.get_for_model(Device).first()` and `Status.objects.get_for_model(Location).first()` in this test for correctness.